18F/identity-idp
# LG-8894: Describe personal key example for assistive technology

## 🎫 Ticket

[LG-8894](https://cm-jira.usa.gov/browse/LG-8894)

## 🛠 Summary of changes

Adds a non-visible text element describing the personal key example shown with placeholder characters, so that a user can understand what is being shown is an example personal key, instead of having "X" announced 16 times.

## 📜 Testing Plan

1. Reset password via "Forgot your password?" on the Sign In screen for a verified account
2. Sign in
3. Click "Reactivate your profile" on the account dashboard
4. Use screen reader to navigate page contents
5. Observe that the personal key example includes useful content to describe its purpose as a preview example

## 👀 Screenshots

_Before:_

https://user-images.githubusercontent.com/1779930/224330375-6a73856d-193b-492b-aab2-3027d8bfa8df.mov

_After:_

https://user-images.githubusercontent.com/1779930/224330394-aee38f69-490f-457a-97f7-966797a3f484.mov
